app/models/manageiq/providers/vmware/infra_manager/inventory/collector/property_collector.rb

```diff
@@ -7,6 +7,8 @@ def destroy_property_filter(property_filter)
     return if property_filter.nil?
 
     property_filter.DestroyPropertyFilter
+  rescue => err
+    _log.warn("Failed to destroy property filter: #{err}")
```

Comment:
I wonder if we should check the string here...something like:

```suggestion
    _log.warn("Failed to destroy property filter: #{err}") unless err.to_s.include?("has already been deleted")
```

Reply:
Yeah, if I catch only `RbVmomi::Fault` I could do `unless err.fault.kind_of?(RbVmomi::VIM::ManagedObjectNotFound)`